plugins/modules/packaging/os/homebrew.py

```diff
@@ -10,6 +10,10 @@
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import absolute_import, division, print_function
+from ansible.module_utils.six import iteritems, string_types
```

Comment:
Please revert the last change and move these imports back to where they came from. Ansible does not follow pep8 with this regard.

Reply:
@felixfontein  ahh, okay, will do, saw some of the failures, just did a mass pep8 format, I'll revert and then narrow down to fixing the previous test failure with whitespace issue :)